src/components/campaigns/Summary/TimeToGive/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import moment from 'moment'
+import Stat from '../Stat'
+
+const remainingtTimeToGive = function (donationEndTime, unit) {
+  return donationEndTime.diff(moment(), unit)
+}
+
+const TimeToGive = ({ donationEndTime }) => {
+  if (!donationEndTime || remainingtTimeToGive(donationEndTime) <= 0) return <span className='TimeToGive--empty' />
+  const daysLeft = remainingtTimeToGive(donationEndTime, 'days')
+  const plural = daysLeft > 1
+  return (
+    <Stat
+      number={daysLeft === 0 ? 'less than 1' : daysLeft}
+      label={`${plural ? 'days' : 'day'} to give`} />
+  )
+}
+
+TimeToGive.displayName = 'CampaignSummaryTimeToGive'
+
+TimeToGive.propTypes = {
+  donationEndTime: React.PropTypes.object
```

Comment:
It is only available after loading is complete. So either I dont render this component at all while we are loading the information or I dont make it requied. Otherwise you will end up with warning of missing props even when everything is working as expected.